releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,23 @@
 
 [Image][image] | [Sample][sample]
 
+## 2.3.1 - June 25, 2025
+
+### Enhancements
+
+* Upgraded dependent packages. [#122](https://github.com/Azure/AppConfiguration-KubernetesProvider/pull/122)
+
+### Bug Fixes
+
+* Fixed a bug that caused the conditional refresh does not work after controller restart. [#119](https://github.com/Azure/AppConfiguration-KubernetesProvider/issues/119)

```

Comment:
I don't see the "conditional refresh" concept explained anywhere in documentation.